build/props.js

```diff
@@ -82,6 +84,27 @@ const figmatokensSYNC = { 'open-props': { ...figmatokens } }
 const FigmaTokensSync = fs.createWriteStream('../open-props.figma-tokens.sync.json')
 FigmaTokensSync.end(JSON.stringify(figmatokensSYNC, null, 2))
 
+if (!fs.existsSync('../dist'))
+  fs.mkdirSync('../dist')
+
+const JS = fs.createWriteStream('../dist/open-props.js')
+JS.end(`var OpenProps = ${JSON.stringify(toObject(), null, 2)}`)
+
+const ES = fs.createWriteStream('../dist/open-props.module.js')
+ES.end(`export default ${JSON.stringify(toObject(), null, 2)}`)
+
+const CJS = fs.createWriteStream('../dist/open-props.cjs')
+CJS.end(`module.exports = ${JSON.stringify(toObject(), null, 2)}`)
+
+// const UMD = fs.createWriteStream('../dist/open-props.umd.js')
+// UMD.end(`module.exports = ${JSON.stringify(toObject(), null, 2)}`)
+
+const TS = fs.createWriteStream('../dist/open-props.ts')
+TS.end(`export default ${JSON.stringify(toObject(), null, 2)}`)
```

Comment:
the TS file doesn't really provide anything. `.JS` + `.D.TS` is enough


```suggestion
```

Reply:
thanks, i was wondering about this. there's a bit more value in the .ts file than the .d.ts file because it holds the values and the value shows in the autocomplete, as opposed to just offering key completion and presenting the value type. i'll test a bit more and see what the most minimal solution is with great DX 👍🏻

Reply:
> there's a bit more value in the .ts file than the .d.ts file because it holds the values and the value shows in the autocomplete

but typescript cannot import anything material from `.ts` files inside node modules. it looks for `.js` and `.d.ts` files

Reply:
plus, ESM environments require file extensions and/or export maps, and node would not know what to do when trying to import `.ts` files.

meanwhile, if you import a `.js` file, TS will (usually) know to look for `.d.ts` file next to it.